components/modelmeshserving/modelmeshserving.go

```diff
@@ -31,6 +31,25 @@ type ModelMeshServing struct {
 	components.Component `json:""`
 }
 
+func (m *ModelMeshServing) DownloadManifests(platform string) error {
+	// If devflags are set, update default manifests path
+	if len(m.DevFlags.Manifests) != 0 {
```

Comment:
maybe we can leave `model-mesh` out of the `devFlags` for now, till they are done with transition, then we know how they want to use this: it could be one repo , but most likely will be two different one for modelmesh one for monitoring